static/js/formatters-internal.js

```diff
@@ -522,6 +523,28 @@ export function price(fieldValue = {}, locale) {
   return price.toLocaleString(localeForFormatting, { style: 'currency', currency: currencyCode });
 }
 
+/**
+ * Returns a localized price range string for the given price range ($-$$$$) and country code (ISO format)
+ * @param {Object} pricerange The price range from LiveAPI entity
+ * @param {string} countrycode The country code from LiveAPI entity (e.g. profile.address.countryCode)
+ * @return {string} The price range with correct currency symbol formatting according to country code
+ */
+export function priceRange(priceRange, countryCode) {
+  if (!priceRange || !countryCode) {
+    console.warn(`No price range or country code given.`);
+    return '';
+  }
+  const currencyCode = LocaleCurrency.getCurrency(countryCode);
+  if (currencyCode) {
+    const currencySymbol = getSymbolFromCurrency(currencyCode);
+    if (currencySymbol) {
+      return Array(priceRange.length+1).join(currencySymbol); 
```

Comment:
For my own edification, what does the `priceRange` object look like?

Reply:
the given priceRange from entity, if it's defined, is like this: '$', '$$', '$$$', or '$$$$'.

Reply:
So it's a string? The param type listed in the method signature is object.

Reply:
oops, I missed that. Updated!